server/game/gamesteps/actionwindow.js

```diff
@@ -22,9 +22,9 @@ class ActionWindow extends PlayerOrderPrompt {
 
     activePrompt() {
         return {
-            menuTitle: 'Any actions or reactions?',
+            menuTitle: 'Initiate an action, or pass',
             buttons: [
-                { text: 'Done' }
+                { text: 'Pass' }
```

Comment:
Yep. Depending on how much work fixing those tests would be, I might change it back to "Done". It was mostly about not asking for reactions in action windows.